benchmarks/benchmarks/bench_function_base.py

```diff
@@ -98,44 +98,122 @@ def time_select_larger(self):
 class Sort(Benchmark):
     params = [
         ['quick', 'merge', 'heap'],
-        ['float32', 'int32', 'uint32']
+        ['float32', 'int32', 'uint32'],
+        [
+            'ordered',
```

Comment:
These should all be documented as to what they are and what they are trying to test. That is probably more work than writing the tests, but will help interpret the results.

Reply:
Maybe add a big docstring to the class to contain the documentation.

Reply:
Done!